src/windows-emulator/emulator_utils.hpp

```diff
@@ -267,29 +270,30 @@ class emulator_allocator
 	uint64_t active_address_{0};
 };
 
-inline std::wstring read_unicode_string(const emulator& emu, const UNICODE_STRING ucs)
+
+inline std::u16string read_unicode_string(const emulator& emu, const UNICODE_STRING<EmulatorTraits<Emu64>> ucs)
```

Comment:
an overload like this `inline std::u16string read_unicode_string(const emulator& emu, uint64_t ucs)` would be nice, but afaik calling it is ambiguous because of implicit conversion operators in emulator_object. might make sense to make these explicit

Reply:
see #52